tensilelite/Tensile/KernelWriterAssembly.py

```
@@ -11052,6 +11052,44 @@ def bufferLoadImpl(soffset):
           rv.add(BufferLoadB128(dst=dst, vaddr=addr0, saddr=addr1, \
                                 soffset=soffset, mubuf=mubuf2, comment=comment))
           return rv
+        elif bpl==64:
+          rv = Module("emulated _buffer_load_b256")
+          # +0.25
+          dst = None if lds else vgpr(destVgpr, rpv//4)
```

Could we throw an error for the `lds` case instead of setting `dst = None` - so we only handle `blp == 64` and `not lds`. Something like
```
elif bpl==64 and not lds:
  rv = Module("emulated _buffer_load...)
  dst = vgpr(...
```
My concern here is that for the lds case we also need to update `m0` when we have multiple loads.

---

I will work on it right now

---

Ah, I see your point...the code will be cleaner, yes!

---

The code has been corrected according to your suggestions, the compilation process works now, the test conditions as given above are also passing and library is providing expected boost!